zsl/unittest/db.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
```

Comment:
Use all future imports like in http://python-future.org/quickstart.html#if-you-are-writing-code-from-scratch In this instance nothing happens if you omit some.